code/obj/critter/misc.dm

```diff
@@ -1573,3 +1573,53 @@
 
 	CritterAttack(mob/M)
 		src.task = "chasing"
+
+/obj/critter/livingtail
+	name = "Living tail"
+	desc = "A twitching saurian tail, you feel mildly uncomfortable looking at it."
+	icon_state = "twitchytail"
+	density = 1
+	health = 30
+	wanderer = 1
+	firevuln = 1
+	brutevuln = 1
+	butcherable = 0
```

Comment:
????

Reply:
Removed the redundant lines

Reply:
no like why can't i butcher it

Reply:
It despawns instantly upon dying so i didnt think that i needed to make it butcherable. Still, changed it.

Reply:
no like, why can't i butcher it into meat or something, in a feature sense
does it correctly output meat with just the flag enabled, or does the output type need to be adjusted too?

Reply:
The meat works fine and comes as "living tail's" meat. Though realistically its impossible to get for a regular player.